src/Task/ApiGen/ApiGen.php

```diff
@@ -30,6 +31,7 @@ class ApiGen extends BaseTask implements CommandInterface
      * @var string
      */
     protected $command;
+    protected $operation = ' generate';
```

Comment:
I was only mimicking how it's done in `CommandArguments::args()`, but that should be an easy enoguh fix.